src/style.css

```diff
@@ -2,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  padding: 12px 0;
+}
+
+.search-bar {
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  font-weight: 300;
+  transition: border-color 0.3s ease;
+  outline: none;
+  font-size: 16px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-family: inherit;
+}
+
+input::placeholder {
+  color: #3d4e61;
+  font-weight: normal;
+}
+
+[data-qa='big'] input {
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  background-size: 19px;
+  background-position:
+    25px center,
+    0 0;
+}
```

Comment:
The top indent of 20px for the big search bar is missing. Please ensure that there is a top indent of 20px for the big search bar as specified in the task requirements.